Documentation/operations/troubleshooting_clustermesh.rst

```diff
@@ -59,11 +99,15 @@ Manual Verification of Setup
 
  #. Validate that the connection to the remote cluster could be established.
     You will see a log message like this in the ``cilium-agent`` logs for each
-    remote cluster::
+    remote cluster
```

Comment:
Maybe end this with one `:`? Otherwise the grammar is not complete. Same with the next hunk in this patch below.